Apps/BitcoinWallet/BitcoinWallet.vsimporter/BitcoinWallet-WinStore10/Package.appxmanifest

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="e3660482-e8b1-4343-addc-44d336ad21db"
+    Publisher="CN=islandwood"
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="e3660482-e8b1-4343-addc-44d336ad21db" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>BitcoinWallet</DisplayName>
+    <PublisherDisplayName>bbowman</PublisherDisplayName>
```

Comment:
this is my least favorite thing ever.

---
In reply to: [107052264](https://github.com/Microsoft/WinObjC-Samples/pull/19#discussion_r107052264) [](ancestors = 107052264)